native/include/node.h

```diff
@@ -51,6 +51,16 @@ typedef struct mem mem_t;
 /* configuration of the node received from jboss cluster. */
 struct nodemess
 {
+    /* part updated in httpd */
+    int id;                  /* id in table and worker id */
```

Comment:
The comment needs to after the int id;
In fact please just move the int id;

Reply:
I wanted to preserve the structure within the struct, but as you wish, that's not a problem.

Reply:
Done. I also added the note as I did in other cases.